src/frontend/src/utils/enum-pipes.ts

```diff
@@ -68,3 +68,10 @@ export const ChangeRequestStatusTextPipe: (status: ChangeRequestStatus) => strin
       return 'Open';
   }
 };
+
+export function formatEnumValue(value: string) {
```

Comment:
remove this method, now that it's unused

Reply:
Just removed this